src/Components/FormStartPage/FormStartPage.tsx

```diff
@@ -8,6 +8,7 @@ type FormStartPageProps = {
 };
 let FormStartPage = (props: FormStartPageProps) => {
   useEffect(() => {
+    sessionStorage.setItem("userId",JSON.stringify(props.userId))
```

Comment:
Setting User ID in session storage to allow for user to refresh

Reply:
Awesome job error handling here incase the user refreshes the page, that makes for a much better UX